server/core/src/https/errors.rs

```diff
@@ -1,11 +1,72 @@
 //! Where we hide the error handling widgets
 //!
 
-use axum::http::header::ACCESS_CONTROL_ALLOW_ORIGIN;
 use axum::http::{HeaderValue, StatusCode};
+use axum::http::header::ACCESS_CONTROL_ALLOW_ORIGIN;
 use axum::response::{IntoResponse, Response};
-use kanidm_proto::internal::OperationError;
+use axum_htmx::{HxReswap, HxRetarget, SwapOption};
 use utoipa::ToSchema;
+use uuid::Uuid;
+
+use kanidm_proto::internal::OperationError;
+
+use crate::https::middleware::KOpId;
+use crate::https::views::errors::ErrorPartialView;
+use crate::https::views::HtmlTemplate;
+
+/// The web app's top level error type, this takes an `OperationError` and converts it into a HTTP response.
+#[derive(Debug, ToSchema)]
+pub enum HtmxError {
+    /// Something went wrong when doing things.
+    OperationError(Uuid, OperationError),
+    // InternalServerError(Uuid, String),
+}
+
+impl From<(&KOpId, OperationError)> for HtmxError {
```

Comment:
This from syntax makes calling HtmxError really messy because now it's "HtmxError::from((a, b))" all the time (note the double parens.). 

It should just be "HtmxError::new()"